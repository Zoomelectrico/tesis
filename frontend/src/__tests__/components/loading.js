import React from 'react';
import renderer from 'react-test-renderer';
import Loading from '../../components/loading';

test('Loading render corrently', () => {
  const component = renderer.create(<Loading />);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
