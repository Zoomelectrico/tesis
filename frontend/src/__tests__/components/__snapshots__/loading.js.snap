// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading render corrently 1`] = `
<div
  className="p-5"
>
  <div
    className="d-flex justify-content-center"
  >
    <img
      alt="uvote logo"
      className="mb-3"
      src={
        Object {
          "ReactComponent": Object {
            "$$typeof": Symbol(react.forward_ref),
            "render": [Function],
          },
          "__esModule": true,
          "default": "logo-color.svg",
        }
      }
      style={
        Object {
          "height": "125px",
          "width": "125px",
        }
      }
    />
  </div>
  <h2
    className="text-center"
  >
    Cargando...
  </h2>
</div>
`;
